library/src/containers/Schemas/Schema.tsx

```diff
@@ -35,22 +38,31 @@ const renderSchemaProps = (
 export const SchemaComponent: React.FunctionComponent<Props> = ({
   name,
   schema,
+  description,
   exampleTitle,
-  hideTitle,
+  hideTitle = false,
+  toggle = false,
+  toggleExpand = false,
 }) => {
   if (!schema) {
     return null;
   }
+  schema.description = schema.description || description || '';
```

Comment:
But here this is the easiest way to do it, because then, in `SchemaProperties` component we use `TableAccessor`  so passing custom value to it is impossible.